api/src/test/java/com/epam/pipeline/app/AclTestConfiguration.java

```diff
@@ -214,6 +216,12 @@
     @MockBean
     protected PipelineRunDockerOperationManager pipelineRunDockerOperationManager;
 
+    @MockBean
+    protected DtsOperationsApiService mockDtsOperationsApiService;
+
+    @MockBean
+    protected DtsRegistryApiService mockDtsRegistryApiService;
+
```

Comment:
Let's put all test beans to a single place which is better to be `AclTestBeans`.

Reply:
Not applicable. See https://github.com/epam/cloud-pipeline/pull/1588#discussion_r531597823.